frontend/src/auth_config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "domain": "rota-flex-101.eu.auth0.com",
+    "clientId": "NqmZExdF9Q14mgi7p5o5RkPt338B2irO"
```

Comment:
I don't know much about auth0, so just wanted to make sure that it's ok to commit clientId to the repo, i.e. it's not sensitive information?

Reply:
I'm pretty sure the clientId can be here, as in Auth0 you define where requests are allowed to come from. So only requests from our app can authenticate with Auth0. There is a secret client ID which should definitely not be shared. I think this public clientId could allow third party apps to use our API for example.